src/Console/ConfigurationResolver.php

```diff
@@ -37,17 +45,17 @@
     const PATH_MODE_INTERSECTION = 'intersection';
 
     /**
-     * @var bool
+     * @var null|bool
```

Comment:
Property is lazy loaded and should be accessed only via getter, which output is not nullable.